docs/src/main/asciidoc/funqy-amazon-lambda-http.adoc

```diff
@@ -8,14 +8,21 @@ https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 
 include::./attributes.adoc[]
 
-If you like link:funqy-http[Funqy HTTP], you can use it on AWS Lambda.  Quarkus allows you to expose multiple
-Funqy functions through HTTP deployed as one AWS Lambda.
+If you want to allow HTTP clients to invoke on your Funqy functions on AWS Lambda, Quarkus allows you to expose multiple
+Funqy functions through HTTP deployed as one AWS Lambda.  This approach does add overhead over the
+regular Funqy AWS Lambda integration and also requires you to use AWS API Gateway.
 
 include::./status-include.adoc[]
 
 Follow the link:amazon-lambda-http[Amazon Lambda Http Guide].  It walks through using a variety of HTTP
 frameworks on Amazon Lambda, including Funqy.
 
+WARNING: The Funqy HTTP + AWS Lambda binding is not a replacement for REST over HTTP.  Because Funqy
+needs to be portable cross a lot of different protocols and function providers its HTTP binding
+is very minimalistic and you will lose REST features like linking and the ability to leverage
+HTTP features like cache-control and conditional GETs.  You may want to consider using Quarkus's
+JAX-RS, Spring MVC, or Vert.x Web Reactive Route support instead.  They also work with Quarkus and AWS Lambda.
```

Comment:
I would list _JAX-RS, Vert.x Reactive Route and Servlet_ not Spring MVC.
By the way, we didn' have any integration test with Spring MVC so better not document it despite it should works.

Reply:
Disagree on on this point.  We have Spring MVC support in Quarkus for all this stuff.  Who writes Servlets anymore?

Reply:
Servlet is tested but not Spring MVC.
If we document Spring MVC we should test it.
I agree that almost nobody writes servlet anymore ...

Reply:
Our Spring MVC integration is a plugin to Resteasy.  Resteasy is tested.

Reply:
OK then